js/app.jsx

```diff
@@ -850,16 +908,10 @@ var Settings = React.createClass({
 
 var routes = (
   <Route name="app" path="/" handler={App}>
-    <DefaultRoute handler={SearchableCaseverList}/>
-    <Route name="caseversions"       path="/caseversion" handler={SearchableCaseverList}/>
-    <Route name="caseversion_search" path="/caseversion/search/:query" handler={SearchableCaseverList}/>
```

Comment:
We still need this path for permanent link sharing. Same for the following lines.